tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /tutorials/get-started/
+---
+
+This is a guide to creating your first Flutter app. If you know
+how to program, you can complete this tutorial. You don’t need
```

Comment:
"know how to program" seems vague.

Maybe more specifically: familiar with object-oriented code and basic programming concepts such as variables, loops, and conditionals.

Reply:
"If you're familiar with mobile or web development" ?

Reply:
I went with Seth's more specific recommendations.